sympy/physics/control/lti.py

```diff
@@ -0,0 +1,655 @@
+            raise ValueError("Exponent must be an Integer.")
+        if p == 0:
+            return TransferFunction(1, 1, self.var)
+        elif p > 0:
+            num_, den_ = self.num**p, self.den**p
+        else:
+            p = abs(p)
+            num_, den_ = self.den**p, self.num**p
+
+        return TransferFunction(num_, den_, self.var)
+
+    def __neg__(self):
+        return TransferFunction(-self.num, self.den, self.var)
+
+    @property
+    def is_proper(self):
+        """
+        Returns True if degree of the numerator polynomial is less than
+        or equal to degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(b*s**2 + p**2 - a*p + s, b - p**2, s)
+        >>> tf1.is_proper
+        False
+        >>> tf2 = TransferFunction(p**2 - 4*p, p**3 + 3*p + 2, p)
+        >>> tf2.is_proper
+        True
+
+        """
+        return degree(self.num, self.var) <= degree(self.den, self.var)
+
+    @property
+    def is_strictly_proper(self):
+        """
+        Returns True if degree of the numerator polynomial is strictly less
+        than degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+        >>> tf1.is_strictly_proper
+        False
+        >>> tf2 = TransferFunction(s**3 - 2, s**4 + 5*s + 6, s)
+        >>> tf2.is_strictly_proper
+        True
+
+        """
+        return degree(self.num, self.var) < degree(self.den, self.var)
+
+    @property
+    def is_biproper(self):
+        """
+        Returns True if degree of the numerator polynomial is equal to
+        degree of the denominator polynomial, else False.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+        >>> tf1.is_biproper
+        True
+        >>> tf2 = TransferFunction(p**2, p + a, p)
+        >>> tf2.is_biproper
+        False
+
+        """
+        return degree(self.num, self.var) == degree(self.den, self.var)
+
+
+class Series(Basic):
+
+    def __new__(cls, *args, evaluate=False):
+        if not all(isinstance(arg, (TransferFunction, Parallel, Series)) for arg in args):
+            raise TypeError("Unsupported type of argument(s) for Series.")
+
+        obj = super(Series, cls).__new__(cls, *args)
+        obj._var = None
+        for arg in args:
+            if obj._var is None:
+                obj._var = arg.var
+            elif obj._var != arg.var:
+                raise ValueError("All transfer functions should be anchored with the same variable.")
```

Comment:
Do you have a test for this?

Reply:
Yeah!
For Series: https://github.com/sympy/sympy/blob/d6771df476f1c2e30010d5fc38f7c88a6671f297/sympy/physics/control/tests/test_lti.py#L347
For Parallel:
https://github.com/sympy/sympy/blob/d6771df476f1c2e30010d5fc38f7c88a6671f297/sympy/physics/control/tests/test_lti.py#L456